api/src/main/java/org/openmrs/PersonName.java

```diff
@@ -28,8 +28,6 @@
 import org.hibernate.search.annotations.IndexedEmbedded;
 import org.openmrs.api.APIException;
 import org.openmrs.api.db.hibernate.search.LuceneAnalyzers;
-import org.openmrs.layout.name.NameSupport;
-import org.openmrs.layout.name.NameTemplate;
```

Comment:
Did you intend to remove the above two imports?

Reply:
That was mentioned in the task description. Should I revert back the deletions?